src/elm/Logs.elm

```diff
@@ -124,7 +124,7 @@ viewLine : StepNumber -> String -> LogFocus -> Int -> SetLogFocus msg -> Bool ->
 viewLine stepNumber line logFocus lineNumber clickAction shiftDown =
     div [ class "line" ]
         [ div
-            [ Util.testAttribute <| "log-line-" ++ String.fromInt lineNumber
+            [ Util.testAttribute <| String.join "-" [ "log", "line", stepNumber, String.fromInt lineNumber ]
```

Comment:
adding step number to the cypress test tag, to allow for less complicated test setup